src/services/config/geoNetworkClient.ts

```diff
@@ -0,0 +1,21 @@
+import axios, { AxiosError } from 'axios';
```

Comment:
can you move this file under src/config. in future we may have more configs coming in. lets keep all configs in a single place

Reply:
The folder has been moved to the `src` folder